articles/using_synapse_as_an_oauth_server.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Using Synapse as an OAuth 2.0 Server"
+layout: article
+excerpt: Follow these steps to register an OAuth Client and authorize access to Synapse.
+category: howto
+---
+
+<style>
+#image {
+    width: 100%;
+}
+#imageSmall {
+    width: 40%;
+}
+</style>
+
+
+# Registering and linking an OAuth 2.0 Client
+Synapse implements the [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html) protocol, which is an extension of OAuth 2.0.  The Open ID Connect endpoint is `https://repo-prod.prod.sagebase.org/auth/v1`, and the Open ID Configuration (aka the "discovery document") is published in the standard location: [https://repo-prod.prod.sagebase.org/auth/v1/.well-known/openid-configuration](https://repo-prod.prod.sagebase.org/auth/v1/.well-known/openid-configuration).
+
+## Create an OAuth 2.0 Client
+You can create a client programmatically as follows:
+
+
+##### Python
+
+```python
+import synapseclient
```

Comment:
Python code is not PEP-8 compliant or consistent, - snakecase, long lines, etc. Please lint! (except for existing Synapse client function/parameter names)